### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by Vishakha-ks**
Added unit tests to cover the EventController APIs, EventRegistrationController APIs and UserController APIs.

### CODE REVIEW

File: src/main/main.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (haziq-k) [inline src/main/main.iml]: What is the purpose of this file?

File: src/test/test.iml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (haziq-k) [inline src/test/test.iml]: is this file needed?

Comment (Vishakha-ks) [reply src/test/test.iml]: Added to .gitignore ,both the above files, test.iml and main.iml since it's local to every system.

**Review by haziq-k — APPROVED** [review]
Looks good

**Review by haziq-k — APPROVED** [review]

**Review by haziq-k — CHANGES_REQUESTED** [review]
The conflicts were not resolved properly. Please remove the latest commit and resolve the conflicts properly

**Review by haziq-k — APPROVED** [review]

**Pull request merged**